src/docker/local-testnet/Dockerfile

```diff
@@ -0,0 +1,61 @@
+FROM aptos-cli as publish
+
+# Declare source paths relative to Econia repository root.
+ARG MOVE_ROOT=src/move
+ARG ECONIA_ROOT=$MOVE_ROOT/econia
+ARG FAUCET_ROOT=$MOVE_ROOT/faucet
+ARG ACCOUNTS_ROOT=src/docker/local-testnet/accounts
+
+# Copy in files.
+WORKDIR /app/
+COPY $ECONIA_ROOT/Move.toml econia/
+COPY $ECONIA_ROOT/sources/* econia/sources/
+COPY $FAUCET_ROOT/Move.toml faucet/
+COPY $FAUCET_ROOT/sources/* faucet/sources/
+COPY $ACCOUNTS_ROOT accounts/
+
+# Get git, start a local testnet, initialize profiles, publish packages.
+RUN apt-get update && \
+    apt-get install git -y && \
+    rm -rf /var/lib/apt/lists/* && \
+    aptos node run-local-testnet \
+        --test-dir data \
+        --with-faucet \
+        & \
+    sleep 30 && \
+    aptos init \
+        --network local \
+        --private-key-file accounts/econia.key \
+        --profile econia \
+        && \
+    aptos move publish \
+        --assume-yes \
+        --included-artifacts none \
+        --named-addresses econia=$(cat accounts/econia.address) \
+        --override-size-check \
+        --package-dir econia \
+        --profile econia \
+        && \
+    aptos init \
+        --network local \
+        --private-key-file accounts/faucet.key \
+        --profile faucet \
+        && \
+    aptos move publish \
+        --assume-yes \
+        --included-artifacts none \
+        --named-addresses econia_faucet=$(cat accounts/faucet.address) \
+        --override-size-check \
+        --package-dir faucet \
+        --profile faucet
+
+# Copy over chain data only.
+FROM aptos-cli as runtime
+COPY --from=publish /app/data /app/data
```

Comment:
why not have

```
EXPOSE 8080
EXPOSE 8081
```

Reply:
As mentioned on the Dockerfile reference for `EXPOSE`: https://docs.docker.com/engine/reference/builder/#expose

```
The EXPOSE instruction does not actually publish the port.
```

Reply:
Noting here discussion at https://stackoverflow.com/questions/22111060

08ddde3 adds requested `EXPOSE` commands

Reply:
TIL about publish vs expose